README.md

```diff
@@ -4,20 +4,21 @@
 
 <img src="media/cwv-extension-drilldown.png">
 
-This extension measures the three [Core Web Vitals](https://web.dev/vitals) metrics in a way that matches how they're measured by Chrome and reported to other Google tools (e.g. [Chrome User Experience Report](https://developers.google.com/web/tools/chrome-user-experience-report), [Page Speed Insights](https://developers.google.com/speed/pagespeed/insights/), [Search Console](https://search.google.com/search-console/about)).
+This extension measures the three [Core Web Vitals](https://web.dev/articles/vitals) metrics in a way that matches how they're measured by Chrome and reported to other Google tools (e.g. [Chrome User Experience Report](https://developers.google.com/web/tools/chrome-user-experience-report), [Page Speed Insights](https://developers.google.com/speed/pagespeed/insights/), [Search Console](https://search.google.com/search-console/about)).
 
-It supports all of the [Core Web Vitals](https://web.dev/vitals/#core-web-vitals) and leverages the [web-vitals](https://github.com/GoogleChrome/web-vitals) library under the hood to capture:
+It supports all of the [Core Web Vitals](https://web.dev/articles/vitals/#core-web-vitals) and leverages the [web-vitals](https://github.com/GoogleChrome/web-vitals) library under the hood to capture:
 
-* [Largest Contentful Paint](https://web.dev/lcp/)
-* [Cumulative Layout Shift](https://web.dev/cls/)
-* [First Input Delay](https://web.dev/fid/)
+* [Largest Contentful Paint](https://web.dev/articles/lcp)
+* [Cumulative Layout Shift](https://web.dev/articles/cls)
+* [Interaction to Next Paint](https://web.dev/articles/inp)
 
-It also supports the pending INP metric:
-* [Interaction to Next Paint](https://web.dev/inp/)
+It also supports the diagnostic metrics:
+* [Time to First Byte](https://web.dev/articles/ttfb)
+* [First Contentful Paint](https://web.dev/articles/fcp)
 
-Finally, it also supports the diagnostic metrics:
-* [Time to First Byte](https://web.dev/ttfb/)
-* [First Contentful Paint](https://web.dev/fcp/)
+And finally, it temporarily supports the following deprecated metrics:
+
+* [First Input Delay](https://web.dev/articles/fid)
```

Comment:
You list FID last here - which I think is the right call, but it's 4th in both HUDs. Should it be moved down to 6th?

Reply:
Less concerned now it's no longer able to mark it as "deprecated"